### ISSUE TITLE
Latest Windows ARM64 CI build doesn't launch

### ISSUE THREAD

**Issue by strongtz** [moonlight-stream/moonlight-qt#1411]
**Describe the bug**
Latest Windows ARM64 CI build doesn't launch at all.

**Steps to reproduce**
Download [MoonlightPortable-arm64-r2609.zip](https://ci.appveyor.com/api/buildjobs/jb77jb48d1y8irm0/artifacts/MoonlightPortable-arm64-r2609.zip), unzip and run Moonlight.exe. It doesn't launch.

**Client PC details**
 - OS: Windows 11 24H2 26100.1742
 - Moonlight Version: 96fb6ee5e9f4e63abc59d70d28e38f9ba7afe006
 - GPU: Qualcomm(R) Adreno(TM) X1-85 GPU

**Moonlight Logs**

```
00:00:00 - Qt Warning: SetProcessDpiAwarenessContext() failed: 操作成功完成。
Qt's default DPI awareness context is DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2. If you know what you are doing, you can overwrite this default using qt.conf (https://doc.qt.io/qt-6/highdpi.html#configuring-windows).
00:00:00 - SDL Info (0): Compiled with SDL 2.31.0
00:00:00 - SDL Info (0): Running with SDL 2.31.0
00:00:00 - Qt Info: Successfully loaded translation for "zh_CN"
00:00:00 - Qt Warning: QQmlApplicationEngine failed to load component
00:00:00 - Qt Warning: qrc:/gui/main.qml:2:1: 模块“QtQuick.Controls”插件“qtquickcontrols2plugin”未找到
```

**Comment by reloxx13** [comment]
once i had to delete old stuff after copying a new version in a old directory where moonklight was installed, try to run it in a clean folder

**Comment by strongtz** [comment]
I'm indeed running it in a clean folder.

**Comment by strongtz** [comment]
The issue seems to be fixed in [v6.1.0](https://github.com/moonlight-stream/moonlight-qt/releases/tag/v6.1.0)
